recipes/platform.exceptions/all/conanfile.py

```diff
@@ -38,7 +38,7 @@ def _minimum_cpp_standard(self):
         return 20
 
     def requirements(self):
-        self.requires("platform.delegates/0.1.3")
+        self.requires("platform.delegates/0.2.7")
```

Comment:
You did not need to move the recipe to a seperate folder for this change, we prefer having as few as possible for maintenance.

Would you mind using something similar to https://github.com/conan-io/conan-center-index/blob/584be22542cb6ce115ea4cc2f57dcf7b4d5f3b10/recipes/restinio/all/conanfile.py#L51-L54

Reply:
Oh, ok. I'll use this way, thank you!